addons/event/views/event_event_views.xml

```diff
@@ -191,24 +191,15 @@
                                         <div class="o_kanban_record_title o_text_overflow" t-att-title="record.name.value">
                                             <field name="name"/>
                                         </div>
-                                        <div t-if="record.address_id.value" class="d-flex">
-                                            <i class="fa fa-map-marker mt-1 me-1" title="Location"/>
+                                        <div t-if="record.address_id.value" class="d-flex ps-1">
+                                            <i class="fa fa-map-marker mt-1 me-2 text-center" title="Location"/>
                                             <span t-esc="record.address_id.value"/>
                                         </div>
-                                    </div>
-                                    <h5 class="p-0">
-                                        <a name="%(event_registration_action_stats_from_event)d"
-                                           type="action"
-                                           context="{'search_default_expected': True}">
-                                            <t t-esc="record.seats_expected.raw_value"/> Expected attendees
-                                        </a>
-                                        <t t-set="total_seats" t-value="record.seats_reserved.raw_value + record.seats_used.raw_value"/>
-                                        <div  class="pt-2 pt-md-0" t-if="total_seats > 0 and ! record.auto_confirm.raw_value"><br/>
```

Comment:
Note that this uses "auto_confirm", so it should be at least modified in the previous commit, even if just to remove the check ^^